mousestyles/PathFeatures.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import pandas as pd
+import math
+
+from mousestyles import data
+from mousestyles.path_index import path_index
+
+def compute_distances(path_obj):
+	r"""
```

Comment:
It looks like the code is mixing tabs and 4 space indentation which might be why the build is failing. Need to make it 4 spaces consistently.